week-1/exercises/initials.js

```diff
@@ -4,3 +4,9 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first 3 characters of each string in upper case
 // Log the variable in each case
+
+let initial=firstName.substring(0,3).toUpperCase()+
```

Comment:
Can you think of a way to construct the `initial` variable without using `toUpperCase()` three times?

Reply:
yeah , i can concate them all then use toUppercase() one time on the initial . thanku